packages/@webex/plugin-meetings/src/media/properties.ts

```diff
@@ -259,10 +259,25 @@ export default class MediaProperties {
 
         return true;
       }
+      LoggerProxy.logger.warn(
+        `Media:properties#getCurrentConnectionType --> missing localCandidate.protocol, candidateType=${localCandidate.candidateType}`
+      );
 
       return false;
     });
 
+    if (foundConnectionType === 'unknown') {
```

Comment:
question: should we not have tests for this?

Reply:
I've tested it manually e2e - confirmed the log is displayed correctly, I don't care about potential regressions here, because we're planning to remove this whole function soon.